src/intrinsics/prepack/global.js

```diff
@@ -551,6 +551,58 @@ export default function(realm: Realm): void {
     })
   );
 
+  // Helper function that replaces the body of a source function with another source function body.
+  // Note that this function affects un-tracked state, so care must be taken
+  // that this helper function is executed at the right time.
```

Comment:
What is the right time? What is the wrong time?

Reply:
That depends on what one wants. In my scenario, I'll do this replacement before the function in question will be invoked for the first time. (Ideally, I'd just replace the function object itself and be done with it without the need for `__replaceFunctionBody`; but unfortunately, the function is  stored in a way that I can't update the reference to it.)